### DISCUSSION TITLE
Dark mode is always off

### DISCUSSION THREAD

**Discussion by poli27** [OrcaSlicer/OrcaSlicer#618] [General]
Dark mode always turns off and I have to turn it back on why?
![2023-04-01 (2)](https://user-images.githubusercontent.com/118916296/229304441-5531cd65-cda1-4f81-8e88-73b027c940a9.png)
![2023-04-01 (1)](https://user-images.githubusercontent.com/118916296/229304443-bb226c02-8122-421c-8b2a-38b727e780b3.png)
![2023-04-01](https://user-images.githubusercontent.com/118916296/229304444-d4f7c3e2-b5c4-4c7c-a386-3aefb0e0b664.png)

**Comment by poli27** [comment]
?

**Comment by TexZeTech** [comment]
@poli27 
![image](https://user-images.githubusercontent.com/7254540/235218951-c9fa6d20-8fee-4ceb-bb99-c5abe31b22bf.png)

![image](https://user-images.githubusercontent.com/7254540/235217961-4974d916-6fc7-4c6a-a448-87d4fc6468d1.png)


For real though I just put  `"dark_color_mode": "1",` into the OrcaSlicer.conf in the config folder.
